Mathlib/Algebra/Order/Ring/Ordering/Basic.lean

```diff
@@ -184,4 +184,333 @@ theorem isOrdering_iff :
     have := h x y
     have := h x (-y)
     cases (by aesop : x ∈ P ∨ -x ∈ P) <;> simp_all [mem_support]
+
+/-!
+### Supports
+-/
+
+theorem supportAddSubgroup_mono {Q : RingPreordering R} (h : P ≤ Q) :
+    P.supportAddSubgroup ≤ Q.supportAddSubgroup :=
+  fun _ ↦ by aesop (add simp mem_supportAddSubgroup)
+
+theorem support_mono {Q : RingPreordering R} [P.HasIdealSupport] [Q.HasIdealSupport] (h : P ≤ Q) :
+    P.support ≤ Q.support := fun _ ↦ by aesop (add simp mem_support)
+
+/-! ## Order operations -/
+
+section Inf
+variable (P₁ P₂ : RingPreordering R)
+
+instance : Min (RingPreordering R) where
+  min P₁ P₂ := { toSubsemiring := min P₁.toSubsemiring P₂.toSubsemiring }
+
+@[simp]
+theorem toSubsemiring_inf : (P₁ ⊓ P₂).toSubsemiring = P₁.toSubsemiring ⊓ P₂.toSubsemiring := rfl
+@[simp] theorem mem_inf (x : R) : x ∈ P₁ ⊓ P₂ ↔ x ∈ P₁ ∧ x ∈ P₂ := .rfl
```

Comment:
```suggestion
@[simp] theorem mem_inf {x : R} : x ∈ P₁ ⊓ P₂ ↔ x ∈ P₁ ∧ x ∈ P₂ := .rfl
```

Reply:
They are rewriting lemmas, and therefore should have enough explicit arguments to direct a rewrite

Reply:
> is there a link explaining the difference? I had been treating equalities and iffs the same!

Been working on a document explaining this, but as always I am short on time!